docs/advanced_topics/images/focal_points.md

```diff
@@ -27,8 +27,9 @@ Might render HTML like:
 
 In newer browsers, you can use the CSS `attr()` function to read these and set `object-position`. For example:
 
-{force=True}
-```css
+```{code-block} css
+:force:
+
```

Comment:
Unfortunately, the Pygments CSS lexer doesn’t seem to be able to parse this block normally even though it is valid CSS (it doesn’t know how to handle the type declarations for `attr()`). It works just fine (and seems to get correct syntax highlighting when you force it to fall back to “relaxed” parsing mode, though.

Since the docs build here is set to fail on warnings, I set `force` here to tell it the warning on this particular block is OK.